stylesheets/base/global/_spacing.styl

```diff
@@ -6,6 +6,11 @@
   &-a400
     padding: $sizing-400
 
+  &-a800
```

Comment:
I think a800 should always be $sizing-800. We should use a modifier if it needs to change at a breakpoint. The paddings do shrink depending on the screen size as the base font size _should_ shrink on smaller screens.